### ISSUE TITLE
GRIIS Belgium query

### ISSUE THREAD

**Issue by timadriaens** [trias-project/indicators#97]
for the purpose of the action plans on unintentional introductions of Union List species, the Belgian IAS secretariat would like to explore the pathway information in the current GRIIS Belgium version. It would be good to have two files:

- the full GRIIS as a flat table cf. **data_input_checklist_indicators.tsv**
- an extract with only `scientific name`, `canonical name`, `kingdom`, `phylum`, `pathway_level1`, `pathway_level2` cf. **pathways_griis_Belgium.tsv** leaving only the rows for `locality `== "Belgium"

In the process of doing so @damianooldoni noted a few inconsistencies in pathway classifications which were fixed in [this issue](https://github.com/trias-project/unified-checklist/issues/69#issuecomment-1029087727) and [here](https://github.com/trias-project/alien-herpetofauna-belgium/issues/17)

@damianooldoni could you create these two .csv files again after the patching of the inconsistencies please?

**Comment by damianooldoni** [comment]
Yes, sure. The inconsistencies were already solved to update the TrIAS checklist indicators online available 👍 
In particular, the webpage with pathway information is available here: https://trias-project.github.io/indicators/04_indicator_pathways.html

Up to now the  [`data_input_checklist_indicators.tsv`](https://github.com/trias-project/indicators/blob/master/data/interim/data_input_checklist_indicators.tsv) is an interim file (`data/interim`) as it is used for further analysis. As you, @timadriaens, need this file more and more often, I will consider it as an output file so I will move it to `data/output` folder. I will save the `pathways_griis_Belgium.tsv` in `data/output` as well.
